setup.py

```diff
@@ -16,7 +16,7 @@
       install_requires=[
           'PyGithub',
           'PyYAML',
-          'dateutils',
```

Comment:
@ralphbean I tried re adding that and I am getting this 'raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'dateutils' distribution was not found and is required by review-rot'